filebeat/filebeat.yml

```diff
@@ -25,7 +25,9 @@ filebeat.inputs:
   id: my-filestream-id
 
   # Change to true to enable this input configuration.
-  enabled: false
+  enabled: true
+
+  dataset: foo
```

Comment:
This ships data to `logs-foo-default`.

The config changes will be revert later, it is only to show how it would be used in the draft.